app/src/main/java/com/zybooks/pizzaparty/PizzaCalculator.kt

```diff
@@ -1,5 +1,6 @@
 package com.zybooks.pizzaparty
 
+import android.util.Size
 import kotlin.math.ceil
 
 const val SLICES_PER_PIZZA = 8
```

Comment:
Instead of using a constant value for SLICES_PER_PIZZA, change your PizzaCalculator function to accept one more Int parameter for  pizzaSize